cdap-distributions/src/hdinsight/createUiDef.json

```diff
@@ -0,0 +1,13 @@
+{
+    "handler": "Microsoft.HDInsight",
+    "version": "0.0.1-preview",
+    "clusterFilters": {
+        "types": [
+            "HBase"
+        ],
+        "tiers": [
+            "Standard"
+        ],
+        "versions": [ "3.4" ]
```

Comment:
good point... this was copied from their examples.  I'll condense them to one line.  And I should probably convert this and cdap-deploy.json to 2-space... but in separate PR.